### ISSUE TITLE
what is ops_thresh and how do you arrive at the hard coded value for each model? 

### ISSUE THREAD

**Issue by fanhui3** [mlmed/torchxrayvision#151]

**Comment by ieee8023** [comment]
> The ops_threshs is there to normalise the value between 0 and 1.

No, the ops_thresh is to normalize the values so that 0.5 is the decision boundary. So values above 0.5 can be considered positive and less than 0.5 considered negative. See Section 2.5 Model Calibration here: https://arxiv.org/pdf/2111.00595.pdf

Also check out Section 4 Transfer Learning Example where "model.op_threshs = None" to remove this calibration so it doesn't get in the way of fine tuning.

You can see how the thresholds are computed here: https://github.com/mlmed/torchxrayvision/blob/f4505b8d4c352be406b903adea7f37e869090c45/scripts/model_calibrate.py#L198-L202

> p.s. Thank you for releasing such useful work to the world, by the way.

No problem!
